homework/english/Volha_Zharava/lecture1/task1.md

```diff
@@ -0,0 +1,46 @@
+Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+Is your mum working today?
+Why are this boys talking now?
+Where does he live?
+Does their mother cook very well?
+Lili is not rich, she dosen't drive a Mercedes.
+Let´s go out. It isn't raining now.
+I have to go now. It is getting dark
+A young man usually stands at the window.
+Ann usually makes doll dresses herself.
+Does his wife have coffee at the moment?
+Let’s go for a walk. It is not raining.
+Look! The boys are coming home.
+Does Maila like Chinese food?
+My grandmother does not work. She is on pension.
+What is your brother watching now?
+Where do you live?
+Do you play the piano well?
+Where is their brother swimming now?
+What do they eat at breakfast?
+They are swimming at the moment?
+Mike is not wearing a tie today.
+Maila does not watch TV every morning.
+Listen! Blanca is singing in the bathroom.
+Bea sometimes read comics.
+
+Найдите ошибки и исправьте их.
+We do not go to school today.
+What are you doing after school?
+At the moment Peter is working in Russia.
+Does he get a new car?
+He never wears a hat.
+He doesn’t like black coffee.
+We have a good time.
```

Comment:
having
maybe you just missed this.

Reply:
i dont agree. because it can be both times - present simple  and present continious
we have any markers words and it is not clear if we meat reaguraly and have fun or we are meeting and currently having fun right now.

also i have looked on the online dictionary
https://context.reverso.net/%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9-%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%B8%D0%B9/we%27re+having+a+good+time

https://context.reverso.net/%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4/%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9-%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%B8%D0%B9/we+have+a+good+time

Reply:
Cool tha we have so uncertain exemples. after the other thoth i can`t disagree withe you, but still it feels like it should be used in Continuous. imho

Reply:
ok. i will enter both sentenсies.
thank for you review!